sdk/keyvault/azure-security-keyvault-jca/pom.xml

```diff
@@ -151,6 +159,31 @@
                   </excludes>
                 </filter>
               </filters>
+              <relocations>
+                <relocation>
+                  <pattern>org.bouncycastle</pattern>
+                  <shadedPattern>com.azure.security.keyvault.jca.implementation.shaded.org.bouncycastle</shadedPattern>
```

Comment:
When the package is relocated, the source code should also use these relocated packages but I don't see any source code changes in this PR.